<ui version="4.0" >
 <class>BoundingBoxProps</class>
 <widget class="QDialog" name="BoundingBoxProps" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Bounding Box </string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>31</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QDoubleSpinBox" name="doubleBoxX1" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="minimum" >
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QDoubleSpinBox" name="doubleBoxX2" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="minimum" >
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" >
       <widget class="QDoubleSpinBox" name="doubleBoxY1" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="minimum" >
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QDoubleSpinBox" name="doubleBoxY2" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="minimum" >
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Z:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" >
       <widget class="QDoubleSpinBox" name="doubleBoxZ1" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="minimum" >
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QDoubleSpinBox" name="doubleBoxZ2" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="minimum" >
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>71</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonOk" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>accepted()</signal>
   <receiver>BoundingBoxProps</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOk</sender>
   <signal>rejected()</signal>
   <receiver>BoundingBoxProps</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
